### PR TITLE
feat: create StoreBadge component

### PR DESCRIPTION
**Pull request by NikitaCG**

### CODE REVIEW

File: src/components/StoreBadge/StoreBadge.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    [Platform.ANDROID]: {
+        ru: GooglePlayRu,
+        en: GooglePlayEn,
+    },
+};
+
+/**
+ * Component for rendering store badges
+ *
+ * @param platform - platform name
```
Comment (korvin89) [inline src/components/StoreBadge/StoreBadge.tsx:36]: There are a lot of props in object. I think u can skip @param and @return parts

File: src/components/StoreBadge/StoreBadge.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    const lang = i18n.lang as Lang;
+    const iconData = badgeData?.[platform][lang] as string;
+
+    if (!iconData) {
+        return null;
+    }
+
+    if (!href) {
+        return (
+            <div onClick={onClick}>
```
Comment (korvin89) [inline src/components/StoreBadge/StoreBadge.tsx:56]: Lets add role="link" here

**Review by korvin89 — CHANGES_REQUESTED** [review]

File: src/components/StoreBadge/StoreBadge.tsx (hunk 2 above)
Comment (NikitaCG) [reply src/components/StoreBadge/StoreBadge.tsx]: fix

File: src/components/StoreBadge/StoreBadge.tsx (hunk 1 above)
Comment (NikitaCG) [reply src/components/StoreBadge/StoreBadge.tsx]: I drop jsdoc, because eslint requires **@return**

**Comment by korvin89** [comment]
Could u add your name to this [file](https://github.com/gravity-ui/components/blob/main/CODEOWNERS)? :)

**Comment by NikitaCG** [comment]
> Could u add your name to this [file](https://github.com/gravity-ui/components/blob/main/CODEOWNERS)? :)

of course yes, done)

**Review by korvin89 — APPROVED** [review]

**Pull request merged**